src/annotator/frame-observer.js

```diff
@@ -1,16 +1,19 @@
 import debounce from 'lodash.debounce';
 
-import * as FrameUtil from './util/frame-util';
+export const DEBOUNCE_WAIT = 40;
 
 /** @typedef {(frame: HTMLIFrameElement) => void} FrameCallback */
 
-// Find difference of two arrays
-let difference = (arrayA, arrayB) => {
-  return arrayA.filter(x => !arrayB.includes(x));
-};
-
-export const DEBOUNCE_WAIT = 40;
-
+/**
+ * FrameObserver detects iframes added and deleted from the document.
+ *
+ * To enable annotation, an iframe must be opted-in by adding the
+ * `enable-annotation` attribute.
+ *
+ * Eventually we may want annotation to be enabled by default for iframes that
+ * pass certain tests. However we need to resolve a number of issues before we
+ * can do that. See https://github.com/hypothesis/client/issues/530
+ */
```

Comment:
Now would probably be a good time to update `FrameObserver` to not be a default export.

Reply:
Yes, I noticed that. I have done it in a follow up commit.